src/search/module-selector.tsx

```diff
@@ -68,10 +60,10 @@ const ModuleSelectorComponent = ({ app }: ModuleSelectorProps): React.JSX.Elemen
 		) {
 			updateModule((modules.find((m) => m.app === app) ?? modules[0])?.route);
 		} else if (module === undefined && app === SEARCH_APP_ID) {
-			updateModule(pathModuleWithSearch);
-			pushHistory({ route: SEARCH_APP_ID, path: `/${pathModuleWithSearch}` });
+			updateModule(modules[0]?.route);
+			pushHistory({ route: SEARCH_APP_ID, path: `/${modules[0]?.route}` });
```

Comment:
module[0] here is still the first module loaded, not the first module based on the order/priority